stdlib/public/core/StringUTF8Validation.swift

```diff
@@ -1,3 +1,15 @@
+//===--- StringUTF8Validation.swift ---------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```

Comment:
It's 2024 now 😉

Reply:
The contents haven't been changed since 2019 though

Reply:
But it has now with the license change 🤔